lib/dap/filter/ldap.rb

```diff
@@ -0,0 +1,114 @@
+module Dap
+module Filter
+
+require 'openssl'
+
+#
+# Decode LDAP probe response
+#
+class FilterDecodeLdapResponse
```

Comment:
Is there a better name for this and wording for the comment?  Would this really work on any LDAP response?  If it would, then the existing name is fine.